PowerEditor/src/Notepad_plus.rc

```diff
@@ -283,6 +283,8 @@ BEGIN
                 MENUITEM "Current Full File path to Clipboard",    IDM_EDIT_FULLPATHTOCLIP
                 MENUITEM "Current Filename to Clipboard",          IDM_EDIT_FILENAMETOCLIP
                 MENUITEM "Current Dir. Path to Clipboard",         IDM_EDIT_CURRENTDIRTOCLIP
+                MENUITEM SEPARATOR
+                MENUITEM "Marked Text to Clipboard",               IDM_EDIT_MARKEDTOCLIP

```

Comment:
IMO, it's not necessary - that's why the existence of the button.

Reply:
Did you see my comment https://github.com/notepad-plus-plus/notepad-plus-plus/pull/8867#issuecomment-700976991 about macro-ability?

Do we leave it unable to be recorded in a macro?

Or do we give it this type of treatment: https://npp-user-manual.org/docs/searching/#searching-actions-when-recorded-as-macros
This type seems like a lot of effort for such a simple command.

Reply:
OK, it's not so bad to have menu command.
We'll have "Marked Style N text to Clipboard" (N:1..5) &  "All Marked text to Clipboard" 6 commands:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8943
We may make sure the name "Marked Text to Clipboard" is not ambiguous among these commands.

Reply:
BTW, feel free to assign to yourself if you're interested in doing https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8943

Reply:
OK, I will do it, once this one is committed; that way there shouldn't be any conflicts to resolve around MENUITEM in Notepad_plus.rc.

Reply:
>make sure the name "Marked Text to Clipboard" is not ambiguous 

Like the existing _Search_ > _Jump up_ > ... commands, we can use something like _Copy Style #**_N_** Text to Clipboard_